posthog/api/dashboard.py

```diff
@@ -51,21 +52,61 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Dashboard:
         validated_data["created_by"] = request.user
         team = Team.objects.get(id=self.context["team_id"])
         use_template: str = validated_data.pop("use_template", None)
-        dashboard = Dashboard.objects.create(team=team, **validated_data)
 
-        if use_template:
+        if request.data.get("duplicate"):
```

Comment:
Hm, would much prefer to avoid extending `create` here, instead this feature would be much cleaner as a resource POST action, i.e. `POST .../{dashboard_id}/duplicate/`.